modules/update-mirror-repository-oc-mirror.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
```

Comment:
Should we indicate we have 2 alternative options to mirror images, either is fine `Mirroring resources using the oc-mirror plug-in` or `Mirroring images using the oc adm release mirror command`. Otherwise I'm worried some users might follow both sections to repeat the image mirror.

Reply:
I'll add more info to the paragraph that introduces these two options to make it clear to users that they should be making a choice between one or the other, and not doing both.